src/sparsezoo/model/utils.py

```diff
@@ -54,6 +57,49 @@
 SAVE_DIR = os.getenv("SPARSEZOO_MODELS_PATH", CACHE_DIR)
 
 
+class ModelResult(BaseModel):
```

Comment:
I would rather create a new python file for those classes in `src/sparsezoo/model/`. It does not feel right to have them in utils.

Reply:
Okay! approved